hs_core/testing.py

```diff
@@ -502,7 +502,7 @@ def timeseries_metadata_extraction(self):
         # there should be one format element
         self.assertEqual(self.resTimeSeries.metadata.formats.all().count(), 1)
         format_element = self.resTimeSeries.metadata.formats.all().first()
-        self.assertEqual(format_element.value, 'application/sqlite')
+        self.assertEqual(format_element.value, 'application/vnd.sqlite3')
```

Comment:
A new mime type. I haven't seen that happen before!

Reply:
I was surprised!